lib/index.js

```diff
@@ -238,7 +260,7 @@ module.exports.renderSync = function(options) {
 
   if (importer) {
     options.importer = function(file, prev) {
-      return { objectLiteral: importer(file, prev) };
+      return importer(file, prev);
```

Comment:
That got me thinking that we probably don't have to wrap the importer callback at all anymore.